src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -1,55 +1,68 @@
+                                            @if (course.courseIcon) {
+                                                <span class="d-flex align-items-center justify-content-center">
+                                                    <jhi-secured-image [src]="course.courseIcon" />
+                                                </span>
+                                            } @else {
+                                                <div class="course-circle d-flex align-items-center justify-content-center">
+                                                    <span class="fs-4">{{ course?.title | slice: 0 : 1 }}</span>
+                                                </div>
+                                            }
+                                            <div class="h6 fw-normal mb-0 course-title text-wrap">{{ course?.title }}</div>
+                                        </button>
+                                    }
                                 </div>
-                            }
+                            </div>
                             @if (!isNavbarCollapsed) {
-                                <div id="test-course-title" class="h6 mb-0 fw-bold text-body auto-collapse">{{ course?.title }}</div>
+                                <div id="test-course-title" class="course-title h6 mb-0 fw-bold text-body auto-collapse">{{ course?.title }}</div>
                             }
                         </div>
-                        <hr class="mt-0" />
-                        <ul class="navbar-nav justify-content-end flex-grow-1 text-decoration-none">
-                            @for (sidebarItem of sidebarItems; track sidebarItem) {
-                                <li class="nav-item" [hidden]="sidebarItem.hidden">
-                                    @if (sidebarItem.hasInOrionProperty && sidebarItem.showInOrionWindow !== undefined) {
-                                        <ng-template
-                                            *ngTemplateOutlet="navItemOrionFilter; context: { $implicit: sidebarItem, iconTextTemplate: navIconAndText, extraPadding: false }"
-                                        />
-                                    } @else {
-                                        <ng-template *ngTemplateOutlet="navItem; context: { $implicit: sidebarItem, iconTextTemplate: navIconAndText, extraPadding: false }" />
-                                    }
+                        <div>
+                            <hr class="mt-0" />
+                            <ul class="navbar-nav justify-content-end flex-grow-1 text-decoration-none">
+                                @for (sidebarItem of sidebarItems; track sidebarItem) {
+                                    <li class="nav-item" [hidden]="sidebarItem.hidden">
+                                        @if (sidebarItem.hasInOrionProperty && sidebarItem.showInOrionWindow !== undefined) {
```

Comment:
Do we want to have this element when showInOrionWindow is false?

Reply:
i moved this part of code and it isnt written by me 👍